app/src/main/java/org/oppia/app/splash/SplashFragment.kt

```diff
@@ -0,0 +1,30 @@
+package org.oppia.app.splash
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.oppia.app.HomeActivity
+import org.oppia.app.R
+
+/** This SplashFragments navigates to Home page once the app is finished loading completely*/
```

Comment:
Nit: There should be no empty whitespace between Javadocs and classes, Javadocs should end with punctuation, and there should be a space between the last symbol and '*/'. Also, class Javadocs should read like "An object that does X" or "A type for Y". See: https://google.github.io/styleguide/javaguide.html#s7.1-javadoc-formatting.

One way to change this:

```kotlin
/** A fragment that navigates to [HomeActivity] once the app is fully loaded. */
class SplashFragment...
```

Reply:
Ok will follow it.